packages/sdk-components-react-radix/src/collapsible.tsx

```diff
@@ -35,3 +36,41 @@ export const CollapsibleTrigger = forwardRef<
 export const CollapsibleContent: ForwardRefExoticComponent<
   Omit<ComponentPropsWithRef<typeof Content>, "asChild">
 > = Content;
+
+/* BUILDER HOOKS */
+
+const namespace = "@webstudio-is/sdk-components-react-radix";
+
+// For each CollapsibleContent component within the selection,
+// we identify its closest parent Collapsible component
+// and update its open prop bound to variable.
+export const hooksCollapsible: Hook = {
+  onNavigatorUnselect: (context, event) => {
+    for (const instance of event.instanceSelection) {
```

Comment:
btw does this mean if we have a multiselect in navigator, we will open up all instances? :) 
I have a feeling user expectation would be to do that only when user clicked one particular

Reply:
No, it's a path to root

Reply:
so instanceSelection is always one? I think I don't udnerstand since this is an array

Reply:
actually we already use instanceSelector ... seems like a good name

Reply:
Selection is wrong semantics here

Reply:
selector is understood as a path, but selection is a list of things, its not a path

Reply:
honestly instancePath is clearer to me than instanceSelector, because with css selectors and stuff it sometimes means a description of a selector, not an ordered items that represent a path, child-parent relationship

I would honestly rename it everywhere to be a "path"

Reply:
I am being extra nitpicky here because its a public api